### PR TITLE
Tajaran update: the war to end all wars edition

### PR DESCRIPTION
**Pull request by Alberyk**
-changes how the tajara species handles citizenship in game: you now have to pick one of the three factions, this also enables consular officers for them
-adds a new var to the species: default_citizenship, which citizenship should be the default one for that species, so you can have species that can't get biesel citizenship as their main one
-adds a bunch of loadout options to the tajaran section
-restricts tajara from being necropolis contractors

### CODE REVIEW

**Comment by KasparoVy** [comment]
Awesome flavour.

File: code/modules/clothing/gloves/xeno/tajara.dm [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ 	desc = "Green gloves made for Tajaran use."
+ 	species_restricted = list("Tajara")
+
+/obj/item/clothing/gloves/white/tajara
+ 	desc = "White gloves made for Tajaran use."
+ 	species_restricted = list ("Tajara")
+
+/obj/item/clothing/gloves/black/tajara/smithgloves
+	name = "machinist gloves"
+	desc = "Protective leather gloves worn by Adhomian urban workers."
```
Comment (KasparoVy) [inline code/modules/clothing/gloves/xeno/tajara.dm]: Ahdominian*

Comment (Alberyk) [reply code/modules/clothing/gloves/xeno/tajara.dm]: It is Adhomai in auroraverse.

Comment (KasparoVy) [reply code/modules/clothing/gloves/xeno/tajara.dm]: Yep, just trying to wrap my head around the nomenclature there. Ahdomai > Ahdomian, or would it be Ahdomaian? Seems fine though

File: code/modules/clothing/under/accessories/xeno/tajara.dm [hunk 2]
```diff
@@ -85,3 +85,56 @@
...
+	flippable = 1
+	drop_sound = 'sound/items/drop/ring.ogg'
+
+/obj/item/clothing/accessory/dpra_pin
+	name = "almariist pin"
+	desc = "A small, black flag pin worn by government officials of the Democratic People's Republic of Adhomai."
+	icon_state = "pin-ala"
+	item_state = "pin-ala"
+	overlay_state = "pin-ala"
+	description_fluff = "The government of the DPRA seeks to dismantle the ethnic caste system present intajaran society. Instead of creating policies and using the force of the state \
```
Comment (KasparoVy) [inline code/modules/clothing/under/accessories/xeno/tajara.dm]: Present in Tajaran society*

File: code/modules/clothing/under/accessories/xeno/tajara.dm [hunk 3]
```diff
@@ -85,3 +85,56 @@
...
+/obj/item/clothing/accessory/hadii_pin
+	name = "hadiist party pin"
+	desc = "A small, red flag pin worn by members of the Hadiist party."
+	icon_state = "hadii-pin"
+	item_state = "hadii-pin"
+	overlay_state = "hadii-pin"
+	description_fluff = "The Party of the Free Tajara under the Leadership of Hadii is the only and ruling party in the PRA, with its leader always being the elected president. \
+	They follow Hadiism as their main ideology, with the objective of securing the tajaran freedom and place in the galactic community. Membership of the Hadiist Party is not open. \
+	For anyone to become a member, they must be approved by a committee that will consider their qualifications and past. Goverment officials can grant honorary memberships, this is \
+	seem as nothing but a honor and does not grant any status or position that a regular Party member would have."
```
Comment (KasparoVy) [inline code/modules/clothing/under/accessories/xeno/tajara.dm]: This is seen as nothing but an honor*

File: code/modules/clothing/under/accessories/xeno/tajara.dm [hunk 4]
```diff
@@ -85,3 +85,56 @@
...
+	drop_sound = 'sound/items/drop/card.ogg'
+
+/obj/item/clothing/accessory/hadii_pin
+	name = "hadiist party pin"
+	desc = "A small, red flag pin worn by members of the Hadiist party."
+	icon_state = "hadii-pin"
+	item_state = "hadii-pin"
+	overlay_state = "hadii-pin"
+	description_fluff = "The Party of the Free Tajara under the Leadership of Hadii is the only and ruling party in the PRA, with its leader always being the elected president. \
+	They follow Hadiism as their main ideology, with the objective of securing the tajaran freedom and place in the galactic community. Membership of the Hadiist Party is not open. \
```
Comment (KasparoVy) [inline code/modules/clothing/under/accessories/xeno/tajara.dm]: Tajaran*

File: code/modules/clothing/under/accessories/xeno/tajara.dm [hunk 5]
```diff
@@ -85,3 +85,56 @@
...
+	flippable = 1
+	drop_sound = 'sound/items/drop/ring.ogg'
+
+/obj/item/clothing/accessory/nka_pin
+	name = "diplomatic service order"
+	desc = "A medal issued to the diplomats and consular officers under the service of the New Kingdom of Adhomai."
+	icon_state = "pin-nka"
+	item_state = "pin-nka"
+	overlay_state = "pin-nka"
+	description_fluff = "The New Kingdom of Adhomai culture is characterized as traditional; instead of trying to change the tajaran caste system, those divisions are seen as a natural \
```
Comment (KasparoVy) [inline code/modules/clothing/under/accessories/xeno/tajara.dm]: Capitalization of Tajaran/Tajara in this whole blurb

File: code/modules/client/preference_setup/loadout/loadout_xeno/tajara.dm [hunk 6]
```diff
@@ -165,4 +172,11 @@
 	display_name = "honorary party member card"
 	path = /obj/item/clothing/accessory/badge/hadii_card
 	sort_category = "Xenowear - Tajara"
-	whitelisted = list("Tajara", "Zhan-Khazan Tajara", "M'sai Tajara")
\ No newline at end of file
+	whitelisted = list("Tajara", "Zhan-Khazan Tajara", "M'sai Tajara")
+
+/datum/gear/gloves/shumalia_belt
+	display_name = "hammer buckle belt"
+	description = "A leather belt adorned by a hammer shaped, worn by priesthood and worshippers of Shumaila."
```
Comment (KasparoVy) [inline code/modules/client/preference_setup/loadout/loadout_xeno/tajara.dm]: shape*

File: code/modules/clothing/shoes/xeno/tajara.dm [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+	icon = 'icons/obj/tajara_items.dmi'
+	icon_state = "adhomai_shoes"
+	item_state = "adhomai_shoes"
+	body_parts_covered = FEET|LEGS
+	species_restricted = list("Tajara")
+	contained_sprite = TRUE
+	description_fluff = "Today the fashion that dominates Adhomai shares few similarities to the clothing of old Furs, linen, hemp, silk and other such fabrics were traded for \
+	synthetic versions, creating a massive boom in the nylon industry and textile industry in the cities. Jeans, overcoats, army uniforms, parade uniforms, flags, pants, shirts, ties, \
+	suspenders, overalls are now the fashion of every Tajara from Nal'Tor to Kaltir. The protests of \"Old fashion\" supporters usually shatter against how effective and cheap it is to \
+	make the human clothes are. There are a few notable branches, the long-coat and fedora \"gangster style\". Leather jacket wearing \"Greaser\" or the popular amongst females, short \
```
Comment (KasparoVy) [inline code/modules/clothing/shoes/xeno/tajara.dm]: 'The protests of "Old Fashion\" supporters usually can't stand against how undeniably effective and cheap to produce Human clothes are.'

File: code/modules/clothing/suits/xeno/tajara.dm [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+	desc = "A sterile insulated coat made of leather stitched over fur."
+	icon = 'icons/obj/tajara_items.dmi'
+	icon_state = "medcoat"
+	item_state = "medcoat"
+	icon_open = "medcoat_open"
+	icon_closed = "medcoat"
+	contained_sprite = TRUE
+	description_fluff = "Today the fashion that dominates Adhomai shares few similarities to the clothing of old Furs, linen, hemp, silk and other such fabrics were traded for \
+	synthetic versions, creating a massive boom in the nylon industry and textile industry in the cities. Jeans, overcoats, army uniforms, parade uniforms, flags, pants, shirts, ties, \
+	suspenders, overalls are now the fashion of every Tajara from Nal'Tor to Kaltir. The protests of \"Old fashion\" supporters usually shatter against how effective and cheap it is to \
```
Comment (KasparoVy) [inline code/modules/clothing/suits/xeno/tajara.dm]: 'The protests of "Old Fashion" supporters usually can't stand against how undeniably effective and cheap to produce Human clothes are.' -- same for all other items sharing this description

File: code/modules/background/citizenship/tajara.dm [hunk 9]
```diff
@@ -28,11 +28,92 @@
...
+
+	glasses = null
+	uniform = /obj/item/clothing/under/tajaran/consular/dpra
+	head = /obj/item/clothing/head/tajaran/consular/dpra
+	backpack_contents = list(
+		/obj/item/storage/box/hadii_manifesto = 1,
+		/obj/item/gun/projectile/pistol/adhomai = 1,
+		/obj/item/storage/field_ration = 1
+	)
+	accessory = /obj/item/clothing/accessory/dpra_pin
```
Comment (Nerrathiel) [inline code/modules/background/citizenship/tajara.dm]: Hello, 

is it intentional that the DPRA Consular gets a Hadii manifesto box?

File: code/modules/clothing/head/xenos/tajara.dm [hunk 10]
```diff
@@ -3,8 +3,8 @@
 	contained_sprite = TRUE
 	description_fluff = "Today the fashion that dominates Adhomai shares few similarities to the clothing of old Furs, linen, hemp, silk and other such fabrics were traded for \
 	synthetic versions, creating a massive boom in the nylon industry and textile industry in the cities. Jeans, overcoats, army uniforms, parade uniforms, flags, pants, shirts, ties, \
-	suspenders, overalls are now the fashion of every Tajara from Nal'Tor to Kaltir. The protests of \"Old fashion\" supporters usually shatter against how effective and cheap it is to \
-	make the human clothes are. There are a few notable branches, the long-coat and fedora \"gangster style\". Leather jacket wearing \"Greaser\" or the popular amongst females, short \
+	suspenders, overalls are now the fashion of every Tajara from Nal'Tor to Kaltir. The protests of \"Old fashion\" supporters can't stand against how undeniably effectove and cheap \
```
Comment (KasparoVy) [inline code/modules/clothing/head/xenos/tajara.dm]: Lol, I'm so sorry dude -- I don't mean to nag you but now it says `effectove` everywhere that shares this description (7 instances)

File: code/game/objects/items/tajara.dm [hunk 11]
```diff
@@ -0,0 +1,90 @@
...
+
+/obj/item/tajcard/proc/set_card(var/figure)
+	if(!figure)
+		return
+
+	icon_state = "tajcig_[figure]"
+
+	switch(figure)
+
+		if("haddi")
```
Comment (NonQueueingMatt) [inline code/game/objects/items/tajara.dm]: haddi is nonexistant

**Comment by KasparoVy** [comment]
Cheers man

File: code/modules/background/citizenship/tajara.dm [hunk 12]
```diff
@@ -28,11 +28,91 @@
...
+	voluntarily turn over power to civilian governments, the DPRA's future faces many fundamental changes."
+	consular_outfit = /datum/outfit/job/representative/consular/dpra
+
+/datum/citizenship/dpra/get_objectives(mission_level, var/mob/living/carbon/human/H)
+	var/rep_objectives
+
+	switch(mission_level)
+		if(REPRESENTATIVE_MISSION_HIGH)
+			rep_objectives = pick("Collect evidence of NanoTrasen being unfair or oppressive against Tajaran employees, to be used as leverage in future diplomatic talks",
+							"Convince [rand(1,3)] republican citizen to adopt Democratic People's Republic's citizenship")
```
Comment (Geevies) [inline code/modules/background/citizenship/tajara.dm]:
```suggestion
							"Convince [rand(1,3)] republican citizen(s) to adopt Democratic People's Republic's citizenship")
```

File: code/modules/background/citizenship/tajara.dm [hunk 13]
```diff
@@ -28,11 +28,91 @@
...
+	the Azunja dynasty finds itself struggling to function with their limited constitutional powers and factional in-fighting between the military and the civilian government."
+	consular_outfit = /datum/outfit/job/representative/consular/nka
+
+/datum/citizenship/nka/get_objectives(mission_level, var/mob/living/carbon/human/H)
+	var/rep_objectives
+
+	switch(mission_level)
+		if(REPRESENTATIVE_MISSION_HIGH)
+			rep_objectives = pick("Collect evidence of NanoTrasen being unfair or oppressive against Tajaran employees, to be used as leverage in future diplomatic talks",
+							"Convince [rand(1,3)] republican citizen to adopt New Kingdom's citizenship")
```
Comment (Geevies) [inline code/modules/background/citizenship/tajara.dm]:
```suggestion
							"Convince [rand(1,3)] republican citizen(s) to adopt New Kingdom's citizenship")
```

File: code/modules/background/citizenship/tajara.dm [hunk 14]
```diff
@@ -28,11 +28,91 @@
...
+
+		if(REPRESENTATIVE_MISSION_MEDIUM)
+			rep_objectives = pick("Convince [rand(1,3)] foreign citizens to sign the imperial volunteer Alam'ardii corps pledge")
```
Comment (Geevies) [inline code/modules/background/citizenship/tajara.dm]:
```suggestion
			rep_objectives = pick("Convince [rand(1,3)] foreign citizen(s) to sign the imperial volunteer Alam'ardii corps pledge")
```

File: code/modules/client/preference_setup/loadout/loadout_smoking.dm [hunk 15]
```diff
@@ -22,10 +22,14 @@
 	display_name = "cigar case"
 	path = /obj/item/storage/fancy/cigar
 
-/datum/gear/cigarette
+/datum/gear/
```
Comment (Geevies) [inline code/modules/client/preference_setup/loadout/loadout_smoking.dm]:
```suggestion
/datum/gear
```

File: code/modules/clothing/gloves/xeno/tajara.dm [hunk 16]
```diff
@@ -0,0 +1,47 @@
+/obj/item/clothing/gloves/yellow/specialt
+	name = "tajaran electrical gloves"
+	desc = "These gloves will protect the wearer from electric shock. Made special for Tajaran use."
```
Comment (Geevies) [inline code/modules/clothing/gloves/xeno/tajara.dm]:
```suggestion
	desc = "These gloves will protect the wearer from electric shock. Made specially for use by Tajara."
```

File: code/modules/clothing/suits/xeno/tajara.dm [hunk 17]
```diff
@@ -0,0 +1,168 @@
...
+	armor = list(melee = 60, bullet = 50, laser = 20, energy = 10, bomb = 5, bio = 0, rad = 0)
+	siemens_coefficient = 0.35
+	description_fluff = "The Feudal Era of Amohda is famous for the steel swords which became common. Many renowned swordsmen and famous warriors would travel the land fighting duels of \
+	single combat in their quests to become the greatest swordsman. Modern Amohda is a mix between loyalists to the NKA and to the DPRA, with almost universal praise for a return to \
+	traditional culture, yet often violent disagreement about the course of the island's political future. A sizable third party of monarchists which advocate the reestablishment of the \
+	Imperial Amohdan dynasty also exists, fragmenting the monarchist factions on the island and further complicating political violence in the area."
+
+/obj/item/clothing/suit/armor/cuirass
+	name = "cuirass"
+	desc = "A reinforced cuirass used by the forces of the new Kingdom of Adhomai.."
```
Comment (Geevies) [inline code/modules/clothing/suits/xeno/tajara.dm]:
```suggestion
	desc = "A reinforced cuirass used by the forces of the new Kingdom of Adhomai."
```

**Pull request merged**